packages/react-server/src/ReactFlightServer.js

```diff
@@ -1250,6 +1276,21 @@ function flushCompletedChunks(
       }
     }
     importsChunks.splice(0, i);
+
+    // Next comes directives.
```

Comment:
It's interesting that these come after JS but in Fizz they don't. Does it make sense that a `preconnect` or `prefetchDNS` comes after?

It kind of makes sense that CSS is actually lower pri that JS in the client-only scenario, because you need the JS to render but then you can wait for the CSS at the end with suspensey CSS.

But we don't know this is only CSS.